src/github.com/matrix-org/dendrite/roomserver/storage/rooms_table.go

```diff
@@ -11,7 +12,11 @@ CREATE TABLE IF NOT EXISTS rooms (
     -- Local numeric ID for the room.
     room_nid BIGINT PRIMARY KEY DEFAULT nextval('room_nid_seq'),
     -- Textual ID for the room.
-    room_id TEXT NOT NULL CONSTRAINT room_id_unique UNIQUE
+    room_id TEXT NOT NULL CONSTRAINT room_id_unique UNIQUE,
+	-- The most recent events in the room that aren't referenced by another event.
```

Comment:
Oh I see, so the intuition with "latest" is correct, I'm just getting really confused between "references" and "is referenced by".